WebContent/createStudent.jsp

```diff
@@ -1,38 +0,0 @@
-<div id="getForm">
-	<div class="container">
-		<h4 style='color: #000080;'>Enter your full details:</h4>
-		<form class="form-inline" id="myForm" onsubmit="return saveStudent()">
-			<table class="table table-striped">

```

Comment:
Here after don't use < table > --for Forms
Use **Bootstrap Form**   .form-group